addons/stock_account/models/product.py

```diff
@@ -114,11 +109,6 @@ class ProductProduct(models.Model):
     stock_value = fields.Float(
         'Value', compute='_compute_stock_value')
 
-    @api.onchange('type')
-    def onchange_type_valuation(self):
-        # TO REMOVE IN MASTER
-        pass
-
     @api.multi
     def do_change_standard_price(self, new_price, account_id):
         """ Changes the Standard Price of Product and creates an account move accordingly."""
```

Comment:
It would be simpler to simply remove or "group by" the locations.  Do you have knowledge of a lot of people putting different accounts on locations?